tests/unit/test_extras.hxx

```diff
@@ -195,6 +204,18 @@ public:
     output_info.disable();
     mesh->createDefaultRegions();
     output_info.enable();
+
+    // Delete any existing mesh_staggered
+    if (mesh_staggered != nullptr) {
+      delete mesh_staggered;
+      mesh_staggered = nullptr;
+    }
+    mesh_staggered = new FakeMesh(nx, ny, nz);
+    mesh_staggered->StaggerGrids = true;
```

Comment:
Is this line the only difference between `mesh` and `mesh_staggered`?

Reply:
The other (more important?) difference is the `setCoordinates` call just below.